servers/rendering/renderer_rd/shaders/shadow_reduce.glsl

```diff
@@ -6,8 +6,20 @@ VERSION_DEFINES
 
 #define BLOCK_SIZE 8
 
+#ifdef MODE_REDUCE_SUBGROUP
```

Comment:
This looks fishy. It looks like we will use the NVidia thread group sizes whenever we use subgroups.

Reply:
Yeah, I was going to make a version for AMD, but noticed there really is not any noticeable difference in performance so I left it as is. Should try on Intel I guess, but doubt it will make a lot of difference either.

Reply:
Ah okay. I guess it's no problem if it doesn't affect performance.